otherlibs/stdune/src/loc0.ml

```diff
@@ -1,19 +1,133 @@
-include Lexbuf.Loc
+type t =
+  | None_loc
```

Comment:
How about `No_location`? Seems easier to read.

Reply:
Made it `No_loc` to match the module's name